src/registerServiceWorker.ts

```diff
@@ -34,7 +34,9 @@ export default function register() {
     }
 
     window.addEventListener('load', () => {
-      const swUrl = `${process.env.PUBLIC_URL}/service-worker.js`;
+      // workaround for this bug https://github.com/wmonk/create-react-app-typescript/issues/346
```

Comment:
Looks like this is fixed in the latest version of uglifyjs2, will verify and update